ostree-ami-image.sh

```diff
@@ -42,13 +42,25 @@ EDGE_USER_PASSWORD=foobar
 sudo mkdir -p /etc/osbuild-composer/repositories
 
 case "${ID}-${VERSION_ID}" in
-    "rhel-9"*)
+    "rhel-9.3")
         OSTREE_REF="rhel/9/${ARCH}/edge"
         SYSROOT_RO="true"
+        ANSIBLE_OS_NAME="rhel-edge"
```

Comment:
on rhel9.3 and rhel9.4 , the os_name should be redhat?

Reply:
others looks good to me.

Reply:
Fixed, thanks!